Dockerfile

```diff
@@ -34,4 +34,6 @@ LABEL commit=${HAPPY_COMMIT}
 ENV COMMIT_SHA=${HAPPY_COMMIT}
 ENV COMMIT_BRANCH=${HAPPY_BRANCH}
 
-CMD gunicorn --worker-class gevent --workers 8 --bind 0.0.0.0:5000 backend.corpora.api_server.app:app --max-requests 10000 --timeout 30 --keep-alive 5 --log-level info
+# FIXME: REVERT! For now, changed `--workers` from 8 to 1 for now, for TileDB mem requirements on rdev.
```

Comment:
This is probably fine to keep, actually, since worker count used in deployment envs is specified via TF

Reply:
rm'd FIXME, replacing w/comment to justify 1 worker